clusterloader2/state/measurements.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
+
+import (
+	"fmt"
+	"sync"
+)
+
+// MeasurementsState represents state of currently running measurements.
+type MeasurementsState struct {
```

Comment:
This type is public, but doesn't contain any public methods. So why is that public?

Reply:
I've changed structures names so they can be accessed only on package level.
Only TestState, InstancesState will be public.